exts/roles.py

```diff
@@ -0,0 +1,164 @@
+import importlib
+import typing
+
+import naff
+import tansy
+
+import common.utils as utils
+from common.const import METADATA
+
+
+async def check_admin(ctx: naff.Context):
+    return isinstance(ctx.author, naff.Member) and ctx.author.has_permission(
+        naff.Permissions.ADMINISTRATOR
+    )
+
+
+class Roles(naff.Extension):
+    def __init__(self, bot: naff.Client):
+        self.client = bot
+        self.subscribe_name_map = {
+            str(METADATA["roles"]["Changelog pings"]): "Changelog pings",
+            str(METADATA["roles"]["External Changelog pings"]): "External Changelog pings",
```

Comment:
Should we consider adding an option for polls pings?

Reply:
The whole thing is made to be decently dynamic - I'm down with adding more options to it.